extensions/ql-vscode/src/vscode-tests/minimal-workspace/databases.test.ts

```diff
@@ -91,86 +80,82 @@ describe("databases", () => {
     // Unfortunately, during a test it is not possible to convert from
     // a single root workspace to multi-root, so must stub out relevant
     // functions
-    sandbox.stub(workspace, "updateWorkspaceFolders");
-    sandbox.spy(workspace, "onDidChangeWorkspaceFolders");
+    jest
+      .spyOn(workspace, "updateWorkspaceFolders")
+      .mockImplementation(() => true);
   });
 
   afterEach(async () => {
     dir.removeCallback();
     databaseManager.dispose(testDisposeHandler);
-    sandbox.restore();
   });
 
   it("should fire events when adding and removing a db item", async () => {
     const mockDbItem = createMockDB();
-    const spy = sinon.spy();
-    databaseManager.onDidChangeDatabaseItem(spy);
+    const onDidChangeDatabaseItem = jest.fn();
+    databaseManager.onDidChangeDatabaseItem(onDidChangeDatabaseItem);
     await (databaseManager as any).addDatabaseItem(
       {} as ProgressCallback,
       {} as CancellationToken,
       mockDbItem,
     );
 
-    expect((databaseManager as any)._databaseItems).to.deep.eq([mockDbItem]);
-    expect(updateSpy).to.have.been.calledWith("databaseList", [
+    expect((databaseManager as any)._databaseItems).toEqual([mockDbItem]);
+    expect(updateSpy).toBeCalledWith("databaseList", [
       {
         options: MOCK_DB_OPTIONS,
         uri: dbLocationUri().toString(true),
       },
     ]);
-    expect(spy).to.have.been.calledWith({
+    expect(onDidChangeDatabaseItem).toBeCalledWith({
       item: undefined,
       kind: DatabaseEventKind.Add,
     });
 
-    sinon.reset();
```

Comment:
Do we need to clear the calls to `updateSpy` and `onDidChangeDatabaseItem` here? It seems like the `toBeCalledWith` calls will always pass since the calls above already pass. So, I believe this should be something like:

```ts
updateSpy.mockClear();
onDidChangeDatabaseItem.mockClear();
```